### PR TITLE
GETKF C90 case :  merge this PR to PR 413 

### PR DESCRIPTION
**Pull request by metdyn**
## Description

To enable C90 case in GETKF with obs: amsua_metop-b + sondes, we introduced a few changes relative to PR 413. 

The following introduced:
- by pass Eva script
- hard coded directory name:  path_to_ensemble

## Dependencies

None

### CODE REVIEW

**Comment by Dooruk** [comment]
@metdyn , are you saying #413 should be merged before this can be merged?

**Comment by metdyn** [comment]
> @metdyn , are you saying #413 should be merged before this can be merged?
I had an error. 
This PR meges first. 
#413 depends on this PR.

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/model/background_ensemble.yaml [hunk 1]
```diff
@@ -5,9 +5,14 @@ members from template:
     filetype: cube sphere history
     provider: geos
     datapath: '.'
-    filename: 'geos.mem%mem%.%yyyy%mm%dd_%hh%MM%ssz.nc4'
-    state variables: [ua,va,t,ps,delp,q,qi,ql,qr,qs,o3ppmv,phis,frocean,frlake,frseaice,
-                      sheleg,ts,soilt,soilm,u10m,v10m]
+    filenames: ['geos.mem%mem%.%yyyy%mm%dd_%hh%MM%ssz.nc4',
+                'fv3-jedi/bkg/geos.crtmsrf.{{horizontal_resolution}}.nc4']
+    #state variables: [ua,va,t,ps,delp,q,qi,ql,qr,qs,o3ppmv,phis,frocean,frlake,frseaice,
+    #                  sheleg,ts,soilt,soilm,u10m,v10m]
```
Comment (Dooruk) [inline src/swell/configuration/jedi/interfaces/geos_atmosphere/model/background_ensemble.yaml]: Can you erase the commeted out part please?

File: src/swell/suites/localensembleda/flow.cylc [hunk 2]
```diff
@@ -191,13 +191,15 @@
...
-        {%- endfor %}
+        script = true
+#        EnKF not ready to use Eva
+#        script = "swell task EvaObservations $config -d $datetime -m {{model_component}}"
+#        platform = {{platform}}
+#        execution time limit = {{scheduling["EvaObservations"]["execution_time_limit"]}}
+#        [[[directives]]]
+#        {%- for key, value in scheduling["EvaObservations"]["directives"][model_component].items() %}
+#            --{{key}} = {{value}}
+#        {%- endfor %}
```
Comment (Dooruk) [inline src/swell/suites/localensembleda/flow.cylc:202]: I see you had some EVA issues, for now it's fine, we could skip it

File: src/swell/test/suite_tests/localensembleda-tier1.yaml [hunk 3]
```diff
@@ -18,12 +18,13 @@ models:
     local_ensemble_save_posterior_mean: false
     local_ensemble_save_posterior_ensemble_increments: true
     ensmean_only: false
+    path_to_ensemble: /discover/nobackup/rtodling/SwellExperiments/R2D2DataStore/Shared/geos/atmens/c90/Y%Y/M%m/D%d/H%H/geos*%Y%m%d_%H%M%Sz.nc4
```
Comment (Dooruk) [inline src/swell/test/suite_tests/localensembleda-tier1.yaml]: I moved those files to the test location, so just a minor change here, we use a certaun resolution nomenclature (c90 -> 91, c12 -> 13) so let's stick with that for now:

```suggestion
    path_to_ensemble: /discover/nobackup/projects/gmao/advda/SwellTestData/letk/ensemble/91/Y%Y/M%m/D%d/H%H/geos*%Y%m%d_%H%M%Sz.nc4
```

**Review by Dooruk — CHANGES_REQUESTED** [review]
All good, just two minor changes required

File: src/swell/suites/localensembleda/flow.cylc (hunk 2 above)
Comment (metdyn) [reply src/swell/suites/localensembleda/flow.cylc:202]: Sure!

File: src/swell/configuration/jedi/interfaces/geos_atmosphere/model/background_ensemble.yaml (hunk 1 above)
Comment (metdyn) [reply src/swell/configuration/jedi/interfaces/geos_atmosphere/model/background_ensemble.yaml]: Done

File: src/swell/test/suite_tests/localensembleda-tier1.yaml (hunk 3 above)
Comment (metdyn) [reply src/swell/test/suite_tests/localensembleda-tier1.yaml]: Resolved

**Review by Dooruk — APPROVED** [review]

**Pull request merged**